data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg

```diff
@@ -0,0 +1,589 @@
+#textdomain wesnoth-wof
+
+[scenario]
+    id=01_The_Hunt
+    name= _ "The Hunt"
+    next_scenario=02_The_Raid
+    map_file=01_The_Hunt.map
+    {DEFAULT_SCHEDULE_DAWN}
+    turns=unlimited
+    victory_when_enemies_defeated=no
+    carryover_percentage=0
+    {INTRO_AND_SCENARIO_MUSIC revelation.ogg northern_mountains.ogg}
+    {EXTRA_SCENARIO_MUSIC legends_of_the_north.ogg}
+    {EXTRA_SCENARIO_MUSIC vengeful.ogg}
+    {EXTRA_SCENARIO_MUSIC breaking_the_chains.ogg}
+
+    [story]
+        [part]
+            story= _ "From the silent skies above, we of the glider caste have watched."
+        [/part]
+        [part]
+            story= _ "We witnessed the vast isles of Morogor slipping into the sea. The diminishment of our nourishing lands. The cries of starving hatchlings before the Wasting’s forever hush. Newer flights scraped out a meager existence on islets of rock their grandsires would have scorned. With envious eyes they gazed upon the green expanses of the central islands. Then it began..."
```

Comment:
> The cries of starving hatchlings before the Wasting’s forever hush.

Even acknowledging the distinctive, archaic speech of the drakes, this was the one line I couldn't decipher. What is being conveyed here, in contemporary English? (Hush?)

Reply:
"forever hush" is death, seems pretty clear to me.

"wasting" is in the dictionary, and Wikipedia says it's a medical term which fits here. I looked it up to check it wasn't a synonym for tuberculosis, and it's not.

Reply:
Yes, I guessed it was meant to be a synonym for death, but the phrasing just sounds awkward to me. As I said, it's the one archaic line that I had trouble interpreting.

Reply:
Maybe it's because "forever" is not an adjective? Would "eternal hush" be less confusing?

Reply:
The Wasting's eternal hush would make more sense for me.

Reply:
Sounds good, I will change forever to eternal.

Reply:
"forever" changed to "eternal"